### PR TITLE
feature - orders screen

### PR DESCRIPTION
**Pull request by at-tanhle**

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/View/Cell/OrdersCell.swift [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+class OrdersCell: UITableViewCell {
+
+    @IBOutlet weak var priceLabel: UILabel!
+    @IBOutlet weak var quantityLabel: UILabel!
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var deliveryTimeLabel: UILabel!
+    override func awakeFromNib() {
+        super.awakeFromNib()
+
```
Comment (at-daiho) [inline MyApp/View/Cell/OrdersCell.swift]: WARNING - Lines should not have trailing whitespace.

File: MyApp/View/Cell/OrdersCell.swift [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+class OrdersCell: UITableViewCell {
+
+    @IBOutlet weak var priceLabel: UILabel!
+    @IBOutlet weak var quantityLabel: UILabel!
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var deliveryTimeLabel: UILabel!
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        
+    }
```
Comment (at-daiho) [inline MyApp/View/Cell/OrdersCell.swift]: WARNING - No empty line between two open or close brackets.

File: MyApp/View/Controllers/Orders/OrdersViewController.swift [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+//
+
+import UIKit
+
+class OrdersViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+
```
Comment (at-daiho) [inline MyApp/View/Controllers/Orders/OrdersViewController.swift]: WARNING - Limit vertical whitespace to a single empty line. Currently 2.

Comment (at-daiho) [inline MyApp/View/Controllers/Orders/OrdersViewController.swift]: WARNING - Lines should not have trailing whitespace.

**Comment by at-daiho** [comment]
Total linter violations in pull request: 4
Serious: 0
Warnings: 4

File: MyApp/ViewModel/Orders/OrdersViewModel.swift [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    func viewModelForItem(at indexPath: IndexPath) -> OrdersCellViewModel {
+        return OrdersCellViewModel(name: "Gel Lột Mụn Bùn Tinh Chất Than Tre Hoạt Tính", quantity: 5, price: "500.000")
+    }
+
+    func viewModelForHeader(inSection section: Int) -> OrdersSectionHeaderViewModel {
+        return OrdersSectionHeaderViewModel(number: 3, id: 8)
+    }
+
+    func viewModelForFooter(inSection section: Int) -> CartSectionFooterViewModel {
+        return CartSectionFooterViewModel(total: 500000)
```
Comment (at-daiho) [inline MyApp/ViewModel/Orders/OrdersViewModel.swift]: WARNING - Underscores should be used as thousand separator in large decimal numbers.

**Comment by at-daiho** [comment]
Total linter violations in pull request: 1
Serious: 0
Warnings: 1

File: MyApp/View/Controllers/PageMenu/PageMenuViewController.swift [hunk 5]
```diff
@@ -74,3 +78,10 @@ class PageMenuViewController: ViewController {
         navigationController?.pushViewController(vc, animated: true)
     }
 }
+
+// MARK: - CAPSPageMenuDelegate
+extension PageMenuViewController: CAPSPageMenuDelegate {
+    func willMoveToPage(_ controller: UIViewController, index: Int) {
```
Comment (at-tuanmai2) [inline MyApp/View/Controllers/PageMenu/PageMenuViewController.swift]: change func name

File: MyApp/View/Controllers/Overview/OverviewViewController.swift [hunk 6]
```diff
@@ -29,6 +30,12 @@ class OverviewViewController: ViewController {
         configCollectionView()
     }
 
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        addToCartBottomConstraint.constant = viewModel.bottomConstraint
```
Comment (at-tuanmai2) [inline MyApp/View/Controllers/Overview/OverviewViewController.swift]: nên chuyển vào hàm viewDidLayoutSubviews

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

**Pull request merged**